testing/fs-uae/APKBUILD

```diff
@@ -16,13 +16,19 @@ build() {
 	export CC=clang
 	export CXX=clang++
 	unset CFLAGS CXXFLAGS
+	if [ "$CARCH" == "ppc64le" ]; then
```

Comment:
should avoid the bash specific `==`. Eg. `if [ "$CARCH" = "ppc64le" ]`.
But we may want enable this for the other disabled arches as well, so I'd prefer something like:
````sh
case "$CARCH" in
ppc64le) ....
esac
````

Because then its easier to add the other arches: `ppc64le|armhf|aarch64) ...`